.ycm_extra_conf.py

```diff
@@ -115,8 +113,4 @@ def Settings(**kwargs):
     final_flags = [flag for flag in compilation_info.compiler_flags_ if not startswith_any(flag, ignored_flags)]
     final_flags.extend(serenity_flags)
 
-    return {
-        'flags': final_flags,
-        'include_paths_relative_to_dir': DIR_OF_THIS_SCRIPT,
-        'override_filename': filename
-    }
+    return {'flags': final_flags, 'include_paths_relative_to_dir': DIR_OF_THIS_SCRIPT, 'override_filename': filename}
```

Comment:
Would lines like this stay on multiple lines if we added a trailing comma after `'override_filename': filename`? I personally find multiple lines better here, would be nice if black could keep that formatting